web-server/v0.4/src/pages/ComparisonSelect/index.js

```diff
@@ -266,7 +264,7 @@ class ComparisonSelect extends ReactJS.Component {
     }
 
     return (
-      <PageHeaderLayout title={selectedController}>
+      <PageHeaderLayout title={selectedControllers.join(', ')}>
```

Comment:
How many controllers can be selected?  If there are many, how useful is the title?

Could we instead shorten the list with an elipsis or something?

Reply:
The user can select as many controllers as they choose and the PageHeaderLayout component will wrap the title appropriately. The title may be useful to confirm the controllers the user selected but we should eventually display a controller tag next to each result selected. 

Did you see a different behavior with many controllers selected while testing?

Reply:
I did not try many controllers, but it would be good to know what the expected behavior is and make sure this is worth doing.

Reply:
Perhaps we should open an issue to track this?

Reply:
I've opened issue #1166 to track this behavior.